OWNERS

```diff
@@ -0,0 +1,23 @@
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+
+approvers:
+- mnairn
+- maleck13
+- philbrookes
+- sedroche
+- aidenkeating
+- dimitraz
+- matskiv
+- JameelB
+- trepel
+
+reviewers:
+- mnairn
+- maleck13
+- philbrookes
+- sedroche
+- aidenkeating
+- dimitraz
+- matskiv
+- JameelB
+- trepel
```

Comment:
This is the list of people that reviewers would be selected from see https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md#the-code-review-process (No more spamming everyone on every PR @philbrookes)

New people being added to the team should be added here, and maybe eventually progress to an approver.

Reply:
@mikenairn  could you add me?

Reply:
@austincunningham Sorry, must have missed you when i was looking at the contributors list. I added you to both approvers and reviewers like everyone else for now.